### ISSUE TITLE
Build with OpenVINO Execution Provider Fails with 2020.3, 2020.4

### ISSUE THREAD

**Issue by kenkoyanagi** [microsoft/onnxruntime#5019]
**BUG**

- Building ORT with OpenVINO 2020.3 or 2020.4 as the execution provider results in a failure, despite the build page recommending 2020.4. 
- Successful building with the minimum supported version (2020.2). 

**Urgency**
TBD

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10 1903 & 1909
- ONNX Runtime installed from (source or binary): Source
- ONNX Runtime version: Pulled from Master on 8/28.
- Python version: 3.6
- Visual Studio version (if applicable): Both 2017 & 2019
- GCC/Compiler version (if compiling from source): CMake 3.15.4
- CUDA/cuDNN version: N/A
- GPU model and memory: Intel Gen11 Iris Plus graphics, and on system with no iGPU.

**To Reproduce**
- Install OpenVINO 2020.3 and 2020.4 from Intel.
- Pull Master, attempt to build with
`build.bat --config RelWithDebInfo --use_openvino <GPU_FP32, GPU_FP16, CPU_FP32>`
- Choose any of the three. 
- Can alsoinclude `--parallel`

**Expected behavior**
- 2020.3: Fails tests. 147 operator tests fail. ERROR STATUS 8
![image](https://user-images.githubusercontent.com/49453354/91930473-c48bcd00-ec95-11ea-95b9-1e7509608053.png)

- 2020.4: Fails with non-descriptive error. ERROR STATUS 1
```
  gmock.vcxproj -> C:\Users\frozone\Documents\onnxruntime\build\Windows\RelWithDebInfo\lib\RelWithDebInfo\gmock.lib
  gtest.vcxproj -> C:\Users\frozone\Documents\onnxruntime\build\Windows\RelWithDebInfo\lib\RelWithDebInfo\gtest.lib
Traceback (most recent call last):
  File "C:\Users\frozone\Documents\onnxruntime\\tools\ci_build\build.py", line 1824, in <module>
    sys.exit(main())
  File "C:\Users\frozone\Documents\onnxruntime\\tools\ci_build\build.py", line 1763, in main
    build_targets(args, cmake_path, build_dir, configs, args.parallel)
  File "C:\Users\frozone\Documents\onnxruntime\\tools\ci_build\build.py", line 920, in build_targets
    run_subprocess(cmd_args, env=env)
  File "C:\Users\frozone\Documents\onnxruntime\\tools\ci_build\build.py", line 428, in run_subprocess
    env=my_env, shell=shell)
  File "C:\Users\frozone\AppData\Local\Programs\Python\Python36\lib\subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['C:\\Program Files\\CMake\\bin\\cmake.EXE', '--build', 'C:\\Users\\frozone\\Documents\\onnxruntime\\\\build\\Windows\\RelWithDebInfo', '--config', 'RelWithDebInfo']' returned non-zero exit status 1.
```

**Screenshots**
See above.

**Additional context**
Full version numbers for the versions tested.
- 2020.4.287
- 2020.3.194

**Additional questions/concerns**
- Will standard builds with OpenVINO EP be hosted for pip install?

**Comment by jywu-msft** [comment]
@suryasidd , @smkarlap  , can you investigate? is this a possible regression on Windows?

**Comment by kenkoyanagi** [comment]
Any updates on this?

**Comment by smkarlap** [comment]
Some fixes were merged in last week that fixes builds with OpenVINO 2020.3 and 2020.4 with **Release** config. All the tests pass with this config.
The fixes for **RelWithDebInfo** config are different and are in progress currently. Please use **Release** config meanwhile.
